components/usage-api/buf.yaml

```diff
@@ -5,5 +5,7 @@ breaking:
 lint:
   use:
     - DEFAULT
+  except:
+    - ENUM_ZERO_VALUE_SUFFIX
```

Comment:
Seems a reasonable exception based on reading [the docs](https://docs.buf.build/lint/rules#enum_zero_value_suffix).